ratelimit/swim.go

```diff
@@ -95,7 +95,8 @@ func (c *clusterLimitSwim) Allow(clearText string) bool {
 	_ = c.local.Allow(s) // update local rate limit
 
 	if err := c.swarm.ShareValue(key, t0); err != nil {
-		log.Errorf("%s clusterRatelimit failed to share value: %v", c.group, err)
+		log.Errorf("clusterRatelimit '%s' disabled, failed to share value: %v", c.group, err)
+		return true // unsafe to continue otherwise
```

Comment:
For the ref: we may implement mode support sometime in the future https://github.com/zalando/skipper/pull/2097